src/main/java/pages/CheckoutPage.java

```diff
@@ -0,0 +1,66 @@
+package pages;
+
+import Constants.Constants;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class CheckoutPage extends BasePage implements Constants {
+    public CheckoutPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public CheckoutPage waitForPageOpened() {
+        try {
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(CHECKOUT_INF_STEP_TWO_OVERVIEW)));
+        } catch (TimeoutException exception) {
+            Assert.fail(String.format("Checkout Page page it's not loaded! Locator: '%s' was not found!", CHECKOUT_INF_STEP_TWO_OVERVIEW));
+        }
+        return this;
+    }
+
+    public float getNumbers(String attribute) {
+        return Float.parseFloat((attribute).replaceAll("[^0-9.]", ""));
+    }
+
+    public CheckoutPage openPage() {
+        driver.get(URL_SOUSE_DEMO + URL_CHECKOUT);
+        return this;
+    }
+
+    public String getCheckoutOverviewLabel() {
+        WebElement actualResult = driver.findElement(By.xpath(CHECKOUT_INF_STEP_TWO_OVERVIEW));
+        return actualResult.getText();
+    }
+
+    public boolean isDisplayedConfirmOrderMessage() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(CHECKOUT_FINISH_MESSAGE));
+        return driver.findElement(Constants.CHECKOUT_FINISH_MESSAGE).isDisplayed();
+    }
+
+    public boolean getItemTotalCost() {
```

Comment:
name for boolean methods should start from prefix 'is'. 
Your method should be named like isItemTotalCostCorrect